hawkbit-runtime/hawkbit-update-server/src/main/resources/application-db2.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+
+# This profile adds basic configurations for a DB2 DB usage.
+# Keep in mind that you need the DB2 driver in your classpath on compile.
+# see https://www.eclipse.org/hawkbit/documentation/guide/runhawkbit.html
```

Comment:
url is no longer valid, would you please change it to `https://www.eclipse.org/hawkbit/guides/runhawkbit/` also in the other places where it might be used?

Reply:
Done. I have also changed the URL for application-mysql.properties and application-mssql.properties